styles.css

```diff
@@ -9,13 +9,140 @@
     height: auto;
     list-style: none;
     box-sizing: border-box;
+    border: none;
   }
 
-  button {
-    border: none;
+  a {
+    color: #000;
+  }
+
```

Comment:
- [ ] OPTIONAL] I advise you to make the HTML scroll behavior smooth, this will make it look more pleasing and feel more enticing 😀.

```
html {
  scroll-behavior: smooth;
}
```